language/testing-infra/e2e-tests/src/versioning.rs

```diff
@@ -33,7 +34,14 @@ impl VersionedTestEnv {
         }
 
         if version_number > 1 {
-            utils::upgrade_df(&mut executor, &dr_account, &mut dr_sequence_number, Some(2));
+            executor.execute_and_apply(
+                dr_account
+                    .transaction()
+                    .sequence_number(1)
+                    .payload(release_1_2_0_writeset())
+                    .sign(),
+            );
+            dr_sequence_number = 2;
```

Comment:
I think it should be fine to not increment the sequence number here, but not sure if there were other issues that you encountered if you didn't bump it?

Reply:
Since this transaction is signed by Diem Root account I suppose its sequence number get bumped by this upgrade trasction?